debian/changelog

```diff
@@ -1,3 +1,13 @@
+flatpak (1.6.2-1endless1) master; urgency=medium
+
+  * Rebase against upstream master (T29218)
```

Comment:
s/master/1.6.2/ right?

Reply:
Ah, right. I think I just copied the same message from before.

Reply:
I decided to say "Debian bullseye" since this changelog is referring to the packaging.